### PR TITLE
Controllers specs

### PR DESCRIPTION
**Pull request by codepantha**
## Changes introduced via this PR: 👇 

- [x] Added RSpec as a gem to the project

- [x] Created a Request spec file for all your controllers and tested the render and HTTP responses.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,58 @@
...
 
-* Deployment instructions
+### Install
+- Run `bundle install` to install all the necessary dependencies
 
-* ...
+### Usage
+- Run `rails s` to expose a local webserver
+- 
+## Authors
```
Comment (bimbolabuari) [inline README.md]: [Optional] Why don't we use Author instead of Authors since it's a solo project? 🤔

**Review by bimbolabuari — CHANGES_REQUESTED** [review]
Hi @codepantherr, 

You have done a good job so far 🚀  

## Highlights 🎉 

-  Uses RSpec gem for testing
- Has `users_spec.rb` and `posts_spec.rb` testing in `spec/requests/` directory

There are some issues that you still need to work on to go to the next project but you are almost there! 🎉 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
- [ ] Kindly test for `show` actions also for `User` and `Post` controllers

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

--------


_Reviewed by  @bimbolabuari_

**Comment by codepantha** [comment]
@bimbolabuari The show action tests are there. Kindly check again 🙏

**Review by Emmyn5600 — APPROVED** [review]
Hi @codepantherr 

STATUS: APPROVED :partying_face: 

Wooow👏, Your project is complete!  There is nothing else to say other than... it's time to merge it. 👏

Congratulations! 🎉 

Optional suggestions

-N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**